index.html

```diff
@@ -23,18 +23,137 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
 <p id="description">I am a Software Developer in training</p>
 <p id="main-p">I can build basic portfolio page, landing page, or website for your business. 
 If you are interested, lets hang out! My socials are down there!</p>
-<img src="Images/Illustration.png" alt="background-img" id="back-img">
 <div class="icon-container">
-<ul>
-    <li><a href="https://github.com/adamteddychang" target="blank"><img src="Images/github.png" alt="github logo" height="20" class="icons"></a></li>
-    <li><a href="#"><img src="Images/linkedin.png" alt="linkedin logo" height="20" class="icons"></a></li>
-    <li><a href="#"><img src="Images/angelist.png" alt="angelist logo" height="20" class="icons"></a></li>
-    <li><a href="https://twitter.com/AdamChang3" target="blank"><img src="Images/twitter.png" alt="twitter logo" height="20" class="icons"></a></li>
-    <li><a href="#"><img src="Images/Medium.png" alt="medium logo" height="20" class="icons"></a></li>
+<ul class="social_icon_ul">
+    <li class="social_icons"><a href="https://github.com/adamteddychang" target="blank"><img src="Images/github.png" alt="github logo" height="20" class="icons"></a></li>
+    <li class="social_icons"><a href="#"><img src="Images/linkedin.png" alt="linkedin logo" height="20" class="icons"></a></li>
+    <li class="social_icons"><a href="#"><img src="Images/angelist.png" alt="angelist logo" height="20" class="icons"></a></li>
+    <li class="social_icons"><a href="https://twitter.com/AdamChang3" target="blank"><img src="Images/twitter.png" alt="twitter logo" height="20" class="icons"></a></li>
+    <li class="social_icons"><a href="#"><img src="Images/Medium.png" alt="medium logo" height="20" class="icons"></a></li>
 </ul>
 
 </div>
 </section>
+<section id="works-container">
+    <h1 class="works-title">My Recent Works</h1>
+    <div class="projects-container">
+        <img src="Images/yoga.png" alt="yoga png" height="237" class="projects-img">
+        <h1 class="projects-title">Multi-Post Stories</h1>
```

Comment:
- [Optional] It would be nice to limit the number of h1 tags in a webpage to just one use other like h2 instead 😉